docs/3.0.0-beta.x/guides/schedule-article.md

```diff
@@ -0,0 +1,80 @@
+# Schedule article
+
+This guide will explain how to create an article schedule system.
+
+## Introduction
+
+This guide will use this [draft system](.draft.md) guide. So first read it to understand the draft/published status and the Article API that we will use in this guide.
```

Comment:
`You should review it first to understand the draft/published status and the Article API that we will use in this guide.